receiver/webhookeventreceiver/req_to_log.go

```diff
@@ -60,3 +67,21 @@ func appendMetadata(resourceLog plog.ResourceLogs, query url.Values) {
 		}
 	}
 }
+
+// append headers as attributes
+func appendHeaders(config *Config, scopeLog plog.ScopeLogs, headers http.Header) {
+	for k := range headers {
+		// Skip the required header used for authentication
+		if k == textproto.CanonicalMIMEHeaderKey(config.RequiredHeader.Key) {
+			continue
+		}
+		scopeLog.Scope().Attributes().PutStr(headerAttributeKey(k), strings.Join(headers.Values(k), ";"))
+	}
+}
+
+// convert given header to snake_case and add "header" as a namespace prefix
+func headerAttributeKey(header string) string {
+	snakeCaseHeader := strings.ReplaceAll(header, "-", "_")
+	snakeCaseHeader = strings.ToLower(snakeCaseHeader)
+	return "header." + snakeCaseHeader
```

Comment:
Any reason to transform to snake case? That can be done later in a processor

Reply:
Was (maybe poorly) trying to conform to https://opentelemetry.io/docs/specs/semconv/general/naming/

Reply:
The issue is that this is input data, not a deterministic piece of data. You can have collision right now such as two headers with the names “content-length” and “Content Length”. It’s better to let processors perform data sanitization.

Reply:
Fair enough. For our use case it would indeed be easy enough to do this in a processor as you say, without baking it in here. What do you think about maintaining the `header.` namespace? I think it would still be useful to differentiate the attributes this way.